Global/JetBrains.gitignore

```diff
@@ -29,6 +29,7 @@
 
 # IntelliJ
 /out/
+.idea/
```

Comment:
That's because there's some files _under_ `.idea` that we should be tracking. If you do: 

```
git add .
git status
```

You should see a list of the files it wants to commit. Contrast that with what the JetBrains documentation recommends and if there's any that stand out, let's update the rules for those specific files...